In [1]:
#R

In [2]:
x <- c(1,3,2)

In [4]:
x

[1] 1 3 2

In [5]:
mean(x)

[1] 2

In [6]:
sd(x)

[1] 1

 Se tienen dos grupos de personas A,B; en ambos grupos se mide el peso (kg) y la talla (cm)
Grupo A: 67,69,68,70
Grupo B: 171,154,148,151,177
¿Cual de los grupos es más variable? NO se puede aplicar la desviación estándar.
Se debe usar el Coeficiente de variabilidad!!

In [19]:
a<-c(67,69,68,70)
b<-c( 171,154, 148,151,177 )

In [20]:
cva<-sd(a)*100/mean(a)
cvb<-sd(b)*100/mean(b)

In [21]:
cva
cvb

[1] 1.884663

[1] 8.083585

In [22]:
sd(a)
sd(b)

[1] 1.290994

[1] 12.9499

Conclusión: El grupo B al que se le midio la estatura, tiene mayor variabilidad; sin
embargo ambos grupos tienen poca variabilidad

In [28]:
#Media recortada
x<-c(1,23,26,27,33,39,51,56,66,73,73,83,84,86,190)
mean(x)
mean(x,trim=0.1)
x<-c(23,26,27,33,39,51,56,66,73,73,83,84,86)
mean(x)

[1] 60.73333

[1] 55.38462

[1] 55.38462

In [29]:
library(readxl)
datos <- read_excel("datos_ejemplo.xlsx",4)

Calcular: media y el coeficiente de variabilidad de la distribución de las tallas
Usando R para la determinación de la media y coeficiente de variabilidad:

In [30]:
mean(TALLA)
sd(TALLA)
cv<-sd(TALLA)*100/mean(TALLA)
cv

[1] 149.4903

[1] 2.388062

[1] 1.597469

La estatura promedio de los niños encuestados es 149.49cm, y de acuerdo al cv, se puede
decir que hay bastante homogeneidad en la distribución de los datos

3. Determine e interprete el valor de la Moda.


In [32]:
#Determinar e interpretar la moda
table(TALLA) # contabiliza la frecuencia de ocurrencia de cada valor de TALLA

TALLA
143.7 143.8 144.7 144.8 145.1 145.7 145.8 145.9   146 146.8   147 147.1 147.2 
    2     1     1     2     1     2     1     1     1     1     1     2     2 
147.8 147.9   148 148.1 148.2 148.9   149 149.1 149.2 149.8 149.9   150 150.1 
    1     4     1     1     3     2     3     3     4     1     5     5     4 
150.2 150.3 150.8 150.9   151 151.1 151.8 151.9   152 152.1 152.2   153 153.1 
    5     1     1     4     3     3     1     2     4     6     2     2     2 
153.2 153.8 
    1     1 

Mo=152.1; por lo tanto la estatura mas frecuente en lo alumnos es de 152.1 cm

In [34]:
tab.talla<-as.data.frame(table(TALLA))


In [35]:
summary(datos)

   ESTUDIANTE      NIVEL_N               PESO          TALLA      
 Min.   : 23.0   Length:93          Min.   :45.0   Min.   :143.7  
 1st Qu.:391.0   Class :character   1st Qu.:47.0   1st Qu.:148.0  
 Median :594.0   Mode  :character   Median :49.0   Median :150.0  
 Mean   :525.8                      Mean   :48.8   Mean   :149.5  
 3rd Qu.:695.0                      3rd Qu.:50.1   3rd Qu.:151.1  
 Max.   :830.0                      Max.   :52.1   Max.   :153.8  
  INGRESO_FAM    Nro_hermanos      GENERO              NSE           
 Min.   :1035   Min.   :0.000   Length:93          Length:93         
 1st Qu.:2267   1st Qu.:0.000   Class :character   Class :character  
 Median :4115   Median :1.000   Mode  :character   Mode  :character  
 Mean   :4039   Mean   :1.398                                        
 3rd Qu.:4798   3rd Qu.:3.000                                        
 Max.   :9416   Max.   :3.000                                        
 opinion causas desnutricion    NIVEL_NC 